00 Base/src/redux/actions/postActions.tsx

```diff
@@ -0,0 +1,28 @@
+import { GET_POST_FROM_API,DELETE_POST,ADD_POST } from './../constants';
+import { getPosts } from '../../myApi/index';
+
+export const getPostFromApi = ( posts: Object[]) => {
+  return {
+    type: GET_POST_FROM_API,
+    payload: posts,
+  };
+};
+export const fetchPosts = () => {
+  return (dispatch) => {
+    return getPosts().then((res) => dispatch(getPostFromApi(res)));
+  };
+};
+
+ export const deletePost = (id: Number) => {
+  return {
+    type: DELETE_POST,
+    payload: id,
+  };
+};
+
+export const addPost = (post: Object) => {
```

Comment:
Please fix this commented code, you can declare an interface to type post, you can't type as Object